src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Types/RoType.cs

```diff
@@ -192,8 +191,9 @@ private RoType[] ComputeInterfaceClosure()
                     }
                 }
             }
-
-            return ifcs.ToArray();
+            RoType[] arr = new RoType[ifcs.Count];
+            ifcs.CopyTo(arr, 0);
```

Comment:
I should have said ` ifcs.CopyTo(arr);`

Reply:
I'll edit it now. But just curious about the reason? Internally, both will go to `CopyTo(array, 0, _count);`

Reply:
There's no functional difference, but it is easier to read.